### PR TITLE
Document usage with waybar

### PR DESCRIPTION
**Pull request by minego**
This project is WONDERFUL and is exactly what I was looking for. I had a bit of a hard time figuring out how to use it with waybar, so I thought it might be helpful to provide an example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,3 +48,25 @@ Monitor sinks: will print `RUNNING` or `NOT RUNNING`
...
+or `modules-right` list.
+
+```
+	"custom/audio_idle_inhibitor": {
+		"format": "{icon}",
+		"exec": "sway-audio-idle-inhibit --dry-print-both-waybar",
+		"return-type": "json",
+		"format-icons": {
+			"output": "󰅶",
+			"none": "󰾪"
```
Comment (ErikReider) [inline README.md:68]: We should add the other ones too, here's what I use in my config:
```json
"output": "",
"input": "",
"output-input": "  ",
"none": ""
```

File: README.md [hunk 2]
```diff
@@ -48,3 +48,25 @@ Monitor sinks: will print `RUNNING` or `NOT RUNNING`
...
 ```
+
+## Waybar Integration
+
+A custom waybar module can be used to display an icon when any application is
+using your mic or playing any audio.
+
+Add the follow section to your `~/.config/waybar/config` file and add
+`custom/audio_idle_inhibitor` to either the `modules-left`, `modules-center`
+or `modules-right` list.
```
Comment (ErikReider) [inline README.md:59]: We should also specify the font used

**Review by ErikReider — CHANGES_REQUESTED** [review]

**Comment by minego** [comment]
Agreed. Would you like to modify the PR, or woyou rather I update it? I'm fine with either.

**Comment by ErikReider** [comment]
Thanks :)

**Comment by minego** [comment]
Thank you!

**Pull request merged**